package.json

```diff
@@ -20,47 +20,47 @@
     "test:node": "mocha"
   },
   "dependencies": {
+    "@babel/core": "^7.1.2",
+    "@babel/preset-env": "^7.1.0",
     "abortcontroller-polyfill": "^1.1.9",
-    "babel-core": "^6.26.3",
-    "babel-preset-env": "^1.7.0",
-    "broccoli-concat": "^3.2.2",
-    "broccoli-merge-trees": "^3.0.0",
+    "broccoli-concat": "^3.7.3",
+    "broccoli-merge-trees": "^3.0.1",
     "broccoli-rollup": "^2.1.1",
     "broccoli-stew": "^2.0.0",
-    "broccoli-templater": "^2.0.1",
-    "ember-cli-babel": "^6.8.2",
-    "node-fetch": "^2.0.0-alpha.9",
-    "rollup-plugin-babel": "^3.0.7",
+    "broccoli-templater": "^2.0.2",
+    "ember-cli-babel": "^7.1.2",
+    "node-fetch": "^2.2.0",
+    "rollup-plugin-babel": "^4.0.3",
     "whatwg-fetch": "^3.0.0"
   },
   "devDependencies": {
-    "broccoli-asset-rev": "^2.4.5",
-    "broccoli-test-helper": "^1.2.0",
-    "chai": "^4.1.2",
+    "broccoli-asset-rev": "^3.0.0",
+    "broccoli-test-helper": "^1.5.0",
+    "chai": "^4.2.0",
     "ember-cli": "~3.0.2",
     "ember-cli-dependency-checker": "^3.0.0",
     "ember-cli-eslint": "^4.2.1",
-    "ember-cli-htmlbars": "^2.0.1",
-    "ember-cli-inject-live-reload": "^1.4.1",
+    "ember-cli-htmlbars": "^3.0.1",
+    "ember-cli-inject-live-reload": "^2.0.1",
     "ember-cli-pretender": "^3.0.0",
-    "ember-cli-qunit": "^4.3.2",
-    "ember-cli-release": "^0.2.9",
+    "ember-cli-qunit": "^4.4.0",
+    "ember-cli-release": "^1.0.0-beta.2",
     "ember-cli-uglify": "^2.0.0",
-    "ember-data": "^2.16.0",
+    "ember-data": "^3.5.0",
     "ember-disable-prototype-extensions": "^1.1.2",
     "ember-load-initializers": "^1.0.0",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-resolver": "^4.0.0",
-    "ember-source": "~3.0.0",
+    "ember-resolver": "^5.0.1",
+    "ember-source": "~3.5.0",
     "ember-source-channel-url": "^1.0.1",
     "ember-try": "^1.1.0",
     "eslint-plugin-ember": "^5.0.0",
-    "eslint-plugin-node": "^5.2.1",
+    "eslint-plugin-node": "^7.0.1",
     "loader.js": "^4.2.3",
     "mocha": "^5.2.0"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10"
+    "node": "8.* || >= 10"
```

Comment:
@buschtoens why did you drop Node 6 support?

Reply:
To quote Jeff Sessions: "I do not recall."

I _think_ it was because one of the dependencies I've updated dropped Node 6 support. I can checkout the PR again later and see, whether this is actually the case and whether we can avoid that one dep to get this out the door.

Reply:
I've rebased and reverted that commit. Let's see, if the Travis goes green. :)